upgrades.rst

```diff
@@ -181,8 +183,22 @@ Once you have a Caddyfile you'll need to run these two commands:
 
 Where $NETMAKER_BASE_DOMAIN is the base domain you used for your Netmaker setup (the part after "dashboard." in your Dockerfile) and $YOUR_EMAIL is your email address.
 
+If users still want to keep using Traefik as the reverse-proxy instead of Caddy for v0.17.0 and above, refer to this docker-compose file https://gist.github.com/alphadose/1602e5dcba500f75ab0b873d4441236b
```

Comment:
would have been nicer if we could host the gist in the org instead. but unfortunately github doesnt support